### PR TITLE
Access Control m-assign-access

### PR DESCRIPTION
**Pull request by TheMkrage**
inside of m panel, there is a command called m-assign-access. It is used like this: m-assign-access <username> <UserType>

Right now, it takes the username and type (which is a enum I added) and sends what is needed to the server. I wrote the method in server to get this request and send it to server’s controller (that is where priyanka comes in) This is tested as it successfully sends and prints the correct data sent on the server side.

### CODE REVIEW

File: src/codeu/chat/common/User.java [hunk 1]
```diff
@@ -38,8 +38,8 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public final class User {
 
-	private HashMap<Uuid, Integer> conversationInterests = new HashMap<>();
-	private ArrayList<Uuid> userInterests = new ArrayList<Uuid>();
+	public HashMap<Uuid, Integer> conversationInterests = new HashMap<>();
```
Comment (KevinWorkman) [inline src/codeu/chat/common/User.java]: Do these need to be public?

File: src/codeu/chat/server/Controller.java [hunk 2]
```diff
@@ -249,4 +250,12 @@ public Time statusUpdate() {
 		return null;
 	}
 
+	// needs to check if user with userId has permission to change the Type of user with username in given convo
+	// if so, update model, if not, return null
+	@Override
+	public Uuid changeAccessControl(Uuid userID, Uuid convo, String username, UserType type) {
+		// TODO Auto-generated method stub
```
Comment (KevinWorkman) [inline src/codeu/chat/server/Controller.java:257]: Is this going to be done in another PR? Or by somebody else?

**Review by KevinWorkman — COMMENTED** [review]
Overall looks good. Just a couple tiny notes below.

One big thing: would it be possible to add tests for the new code?

File: src/codeu/chat/server/Controller.java (hunk 2 above)
Comment (TheMkrage) [reply src/codeu/chat/server/Controller.java:257]: Yes, Priyanka is working on that currently
